concepts/05 Widgets/Scheduler/020 Data Binding/10 Simple Array/15 ArrayStore.md

```diff
@@ -42,11 +42,76 @@ If you want to extend the functionality of a JavaScript array, place it into the
         [dataSource]="appointmentStore">
     </dx-scheduler>
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxScheduler
+            :data-source="dataSource"
+        />
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import DxScheduler from 'devextreme-vue/scheduler';
+    import ArrayStore from 'devextreme/data/array_store';
+
+    const appointments = [ /* ... */ ];
+
+    export default {
+        components: {
+            DxScheduler
+        },
+        data() {
+            return {
+                dataSource: new ArrayStore({
+                    data: appointments,
+                    onLoaded: function () {
+                        // Event handling commands go here
+                    }
+                })
+            }
+        }
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import Scheduler from 'devextreme-react/scheduler';
+    import ArrayStore from 'devextreme/data/array_store';
+
+    const appointments = [ /* ... */ ];
+
+    class App extends React.Component {
+        render() {
+            return (
+                <Scheduler
+                    dataSource={new ArrayStore({
```

Comment:
This is not a pretty way to declare stores. Please look at [examples in the reference](https://js.devexpress.com/Documentation/20_1/ApiReference/Data_Layer/ArrayStore/)